src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -1141,15 +1141,15 @@ namespace SimAirServingZones {
 								}
 							}
 						}
```

Comment:
I commented out the below since a controller in the OA system is now simulated with the coil. This was added just before my correction so I'm not sure it's still needed. If a coil is in the OA system, should it be locked out if economizer is active AND the user told me to lock it out? I think yes.

Reply:
Oh, and I commented these lines out because there was an array bounds exception because AirSysNum exceeded the allocation of OutsideAirSys. My previous argument is just a good reason to keep it out.

Reply:
The fix for this incorrect index variable is what started all of this.  You can grab the pending changes in #6413 if you want.  The check on OA sys HW coils was added by @Nigusse to fix a problem where an HW preheat coil didn't behave the same as an electric.